### PR TITLE
CHORE: update the readme files

### PR DESCRIPTION
**Pull request by phong1233**
# Description

Update the readme to be more clear

## Type of change

Please delete options that are not relevant.

- [ ] BUG (Fixing or documenting a bug)
- [ ] FEAT (Add a new feature to the project)
- [ ] CHORE (Something that needs to be done outside of the issue)
- [x] DOCS (Implement documentation)
- [ ] TASK (A task that must be done to complete an issue)

# Checklist:

- [x] My pull request references the issue in the comments
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,94 @@
...
+* Start Supreme ERP
+    ```
+    docker-compose up
+    ```
+* OR start a single service
+    ```
+    docker-compose up (backend or frontend or database)
+    ```
+* Default credentials: `admin@email.com` for the email and `admin` for the password.
+* Frontend is hosted on http://localhost:3000
```
Comment (sebastien-blain) [inline README.md:74]: Can you make it more explicit that the TA will have to go to localhost:300 to have access to the app

**Review by sebastien-blain — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (phong1233) [reply README.md:74]: Sure

Comment (sebastien-blain) [reply README.md:74]: I know that it is on the frontpage, jsut make sure he clicks on the right one lol

File: README.md [hunk 2]
```diff
@@ -1,54 +1,94 @@
...
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#running-using-docker">Running using docker<a></li>
+      </ul>
+    </li>
+    <li><a href="#more-info">More Information</a></li>
+    <li><a href="#contributors">Contributors</a></li>
+  </ol>
+</details>
```
Comment (tjosepo) [inline README.md:36]: gros flex

**Review by tjosepo — APPROVED** [review]

**Pull request merged**